presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -112,6 +109,43 @@ private DataTypeTest createVarcharDataTypeTest()
                 .addRoundTrip(varcharDataType(), "unbounded");
     }
 
+    @Test
+    public void testPrestoCreatedParametrizedVarcharUnicode()
+            throws Exception
+    {
+        unicodeVarcharDateTypeTest().execute(queryRunner, prestoCreateAsSelect("postgresql_test_parameterized_varchar_unicode"));
+    }
+
+    @Test
+    public void testPostgreSqlCreatedParametrizedVarcharUnicode()
+            throws Exception
+    {
+        unicodeVarcharDateTypeTest().execute(queryRunner, postgresCreateAndInsert("tpch.postgresql_test_parameterized_varchar_unicode"));
+    }
+
+    private DataTypeTest unicodeVarcharDateTypeTest()
+    {
+        String sampleUnicodeText = "攻殻機動隊";
```

Comment:
We don't use non-ascii characters in the code base, so change these to `\u` form